drivers/include/lpsxxx.h

```diff
@@ -36,13 +36,22 @@ extern "C" {
 #include <stdint.h>
 #include "periph/i2c.h"
 
+/**
+ * @defgroup drivers_lpsxxx_config     LPS331AP/LPS25HB/LPS22HB driver compile configuration
+ * @ingroup config_drivers_sensors
+ * @{
+ */
 /**
  * @brief   The sensors default I2C address
  *
  * Default address corresponds to SDO/SA0 pad connected to ground. If SDO/SA0
  * pad is connected to power supply, I2C address is 0x5C.
+ * Refer Section 5.2.1  I2C operation in datasheet(LPS331AP).
```

Comment:
I would point out just the name of the sectionn rather than the number on the datasheet of one particular model, as this driver is for a family of devices:
```suggestion
 * Refer to 'I2C Operation' section on the datasheet
```